<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nemocničný informačný systém</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/icons/server-database.png</normaloff>:/icons/server-database.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="pohladStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="rampaStack">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>546</width>
             <height>383</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0" colspan="2">
             <widget class="QCommandLinkButton" name="hladaniePacientaButton">
              <property name="text">
               <string>Hľadanie pacienta</string>
              </property>
              <property name="description">
               <string>Vyhľadanie pacienta vo zvolenej nemocnici</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QCommandLinkButton" name="vykonajHospitalizaciuButton">
              <property name="text">
               <string>Vykonanie hospitalizácie</string>
              </property>
              <property name="description">
               <string>Vykonanie záznamu o začiatku, alebo konci hospitalizácie</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QCommandLinkButton" name="vytvorPacientaButton">
              <property name="text">
               <string>Vytvorenie pacienta</string>
              </property>
              <property name="description">
               <string>Pridanie nového pacienta do globálneho registra pacientov</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QCommandLinkButton" name="vytvorenieNemocniceButton">
              <property name="text">
               <string>Vytvorenie nemocnice</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QCommandLinkButton" name="zrusenieNemocniceButton">
              <property name="text">
               <string>Zrušenie nemocnice</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QCommandLinkButton" name="hospitalizovaniPacientiButton">
              <property name="text">
               <string>Výpis hospitalizovaných pacientov</string>
              </property>
              <property name="description">
               <string>Výpis pacientov, ktorí sú, alebo boli hospitalizovaní v nemocnici podľa zvoleného kritéria</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QCommandLinkButton" name="podkladyButton">
              <property name="text">
               <string>Podklady pre účtovné oddelenie</string>
              </property>
              <property name="description">
               <string>Vytvorenie podkladov pre účtovné oddelenie</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="rampaLabel">
              <property name="text">
               <string>&lt;h2&gt;Rampa&lt;/h2&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="hladatStack">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="zadanieMenaLabel">
          <property name="text">
           <string>Zadajte meno,
alebo rodné číslo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>hladanieEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="MenoRCEdit" name="hladanieEdit"/>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="hladanieButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Hľadať</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="PacientInfoWidget" name="pacientiHladanieList" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="hospitalizacieStack">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="filterButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Filter</string>
          </property>
          <property name="icon">
           <iconset resource="data.qrc">
            <normaloff>:/icons/view-filter.png</normaloff>:/icons/view-filter.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTabWidget" name="filterTab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="filterTabPage1">
           <attribute name="title">
            <string>Poistenci zvolenej poisťovne</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="poistovnaLabel">
              <property name="text">
               <string>Poisťovňa</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="poistovnaSpin"/>
            </item>
            <item>
             <widget class="QPushButton" name="poistovnaZobrazitButton">
              <property name="text">
               <string>Zobraziť</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="filterTabPage2">
           <attribute name="title">
            <string>Hospitalizovaní od-do</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="odLabel">
              <property name="text">
               <string>Od</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="odEdit">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="doLabel">
              <property name="text">
               <string>Do</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="doEdit">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="odDoZobrazitButton">
              <property name="text">
               <string>Zobraziť</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="pacientiHospitalizacieList" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="podkladyStack"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Subor">
    <property name="title">
     <string>&amp;Súbor</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Pomoc">
    <property name="title">
     <string>&amp;Pomoc</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_Subor"/>
   <addaction name="menu_Pomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Hlavný panel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDesktop"/>
   <addaction name="separator"/>
   <addaction name="actionHladat"/>
   <addaction name="actionHospitalizacie"/>
   <addaction name="actionPodklady"/>
  </widget>
  <widget class="QToolBar" name="nemocnicaToolBar">
   <property name="windowTitle">
    <string>Výber nemocnice</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDesktop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/desktop-launch.png</normaloff>:/icons/desktop-launch.png</iconset>
   </property>
   <property name="text">
    <string>Rampa</string>
   </property>
  </action>
  <action name="actionHladat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/edit-find-user.png</normaloff>:/icons/edit-find-user.png</iconset>
   </property>
   <property name="text">
    <string>Hľadať</string>
   </property>
  </action>
  <action name="actionHospitalizacie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/view_text.png</normaloff>:/icons/view_text.png</iconset>
   </property>
   <property name="text">
    <string>Hospitalizácie</string>
   </property>
  </action>
  <action name="actionPodklady">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/view-statistics.png</normaloff>:/icons/view-statistics.png</iconset>
   </property>
   <property name="text">
    <string>Podklady</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Otvoriť</string>
   </property>
   <property name="toolTip">
    <string>Otvoriť súbor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Uložiť</string>
   </property>
   <property name="toolTip">
    <string>Uložiť súbor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>U&amp;ložiť ako</string>
   </property>
   <property name="toolTip">
    <string>Uložiť ako</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Koniec</string>
   </property>
   <property name="toolTip">
    <string>Ukončiť aplikáciu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;O aplikácii</string>
   </property>
   <property name="toolTip">
    <string>Informácie o aplikácii</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>O &amp;Qt</string>
   </property>
   <property name="toolTip">
    <string>O knižnici Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenoRCEdit</class>
   <extends>QLineEdit</extends>
   <header>MenoRCEdit.h</header>
  </customwidget>
  <customwidget>
   <class>PacientInfoWidget</class>
   <extends>QWidget</extends>
   <header>PacientInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hladaniePacientaButton</tabstop>
  <tabstop>vykonajHospitalizaciuButton</tabstop>
  <tabstop>hospitalizovaniPacientiButton</tabstop>
  <tabstop>podkladyButton</tabstop>
  <tabstop>vytvorPacientaButton</tabstop>
  <tabstop>vytvorenieNemocniceButton</tabstop>
  <tabstop>zrusenieNemocniceButton</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>hladanieEdit</tabstop>
  <tabstop>hladanieButton</tabstop>
  <tabstop>filterButton</tabstop>
  <tabstop>filterTab</tabstop>
  <tabstop>poistovnaSpin</tabstop>
  <tabstop>poistovnaZobrazitButton</tabstop>
  <tabstop>odEdit</tabstop>
  <tabstop>doEdit</tabstop>
  <tabstop>odDoZobrazitButton</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
